Adds essurvey onboarding post

cimentadaj opened the pull request.

cimentadaj commented:
@stefaniebutland I believe I'm doing something wrong because I can't see the post on the netifly preview! I created the post in `.Rmd`, rendered as `.md`, added the YAML header tot he `.md`, add the png's separately as the onboarding post suggests but can't see the post.

stefaniebutland commented:
@cimentadaj I apologize for my long delay in responding. I am temporarily putting most blog post reviews and scheduling on hold until after our unconference. Rest assured your post will get a proper review and be published on our blog. I anticipate getting to this in early June.

cimentadaj commented:
@stefaniebutland oh, don't worry! Have fun at the unconference.

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+---
+slug: "essurvey"
+title: Download data from the European Social Survey
```
Wondering if you could modify the title since what you can do with the package is so much more interesting than "download". Maybe try to refer to types of questions you could address?

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+I never thought that I'd be programming software in my career. I started
+using R little over 2 years now and it's been one of the most important
+decisions in my career. Secluded in a small academic office with no one
+to discuss/interact about my new hobbie, I started searching the web for
```
s/hobbie/hobby/

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+the R community is, it made me want to become a data scientist. So I set
+out to do it. Through out the journey I repeatedly found myself using
+the [European Social Survey](http://www.europeansocialsurvey.org/), a
+really neat dataset that collects informations on attitudes, beliefs and
```
s/informations/information/

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+to discuss/interact about my new hobbie, I started searching the web for
+tutorials and packages. After getting to know how amazing and nurturing
+the R community is, it made me want to become a data scientist. So I set
+out to do it. Through out the journey I repeatedly found myself using
```
s/Through out/Throughout/

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+The package was accepted in CRAN in September 2017 and was well received
+among social scientists.
+
+Over a month ago I submitted the package to rOpensci, intimidated but
```
To make the post more timeless, maybe say the month and year instead of "Over a month ago"

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+------------
+
+I never thought that I'd be programming software in my career. I started
+using R little over 2 years now and it's been one of the most important
```
s/R little/R a little/

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+
+
+<!-- # see for blogpost https://github.com/ropensci/roweb2#contributing-a-blog-post -->
+Introduction
```
Reading this intro gave me a huge smile - your embracing data science and positive community experiences. I hope the community continues to live up to this.

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+Leeper](https://github.com/leeper) and [Nujcharee
+Haswell](https://twitter.com/Nujcharee?ref_src=twsrc%5Egoogle%7Ctwcamp%5Eserp%7Ctwgr%5Eauthor)
+and editorial skills of [Maëlle Salmon](http://www.masalmon.eu/). I am
+indebted to their work. I would also like to thank Wiebke Weber at the
```
Is there a site that Wiebke Weber would appreciate a link to?

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+-----------
+
+Using the `essurvey` package is fairly easy. There are are two main
+families of functions: `import_*` and `show_*`. They each complement
```
s/each complement each other/complement each other/

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+Using the `essurvey` package is fairly easy. There are are two main
+families of functions: `import_*` and `show_*`. They each complement
+each other and allow the user to almost never have to go to the European
+Social Survey (ESS) website. The only scenario where you need to enter
```
omit ESS abbrev here and put it in first para instead

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+each other and allow the user to almost never have to go to the European
+Social Survey (ESS) website. The only scenario where you need to enter
+the ESS website is to validate your email. If you haven't registered,
+create an account at <http://www.europeansocialsurvey.org/user/new> and
```
Please consider condensing the text about email registration & fake email and try to keep it in one paragraph i.e. the section saying "Downloading the ESS data requires validating your email everytime ..." surprised me after thinking you were getting to use case and "Once that’s executed you can delete the previous line..." got confusing.

Can you reword telling reader to `set_email` env variable so you don't have to say delete the line later?

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+call will look for the email automatically, stored as an environment
+variable.
+
+Exploring country data
```
Possibly before the "Exploring country data" section, the post would benefit immensely from an explanation / definitions of the nature, scope and terminology of ESS data. This would help people imagine how they might use your pkg to do cool things. 

More importantly, I found the use of these terms confusing and my reading slowed down as I had to make some inferences - rounds, waves, questions, scales, categories. Definition or explanation of these terms in one paragraph about nature of ESS data allows reader to picture what you are doing. Until I understood clearly what x-axis labels ESS1, ESS2 etc were, I had to go back in the text and code snippets to figure them out.

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+variable.
+
+Exploring country data
+----------------------
```
Exploring country data - Could you make this heading a bit more specific to get at the interesting question you're addressing? 

Exploring countries satisfaction with their government (or something that sounds better than that)

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+Using this information, we can download those specific rounds easily
+with `import_country`.
+
+Note: Since `essurvey 1.0.0` all `ess_*` functions have been deprecated
```
Is it better to put the deprecation note after the code snippet?

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+    import_all_cntrounds("Spain")
+
+Getting back to our example, ESS related datasets have missing
+categories coded differently between questions. For example, questions
```
"coded differently between questions." Relating to my request for definitions above, I didn't understand what this meant. Similar with "scales" here. Had to infer, but readers without survey experience might really have difficulty here

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+"Refusal" coded between `7` and `10`. But questions with scales from
+`1:10` are coded as `77`, `88` and `99`. `recode_missings` accepts a
+`tibble` as a main argument and automatically returns a new `tibble`
+with all missing values recoded as `NA`. You should checkout
```
s/checkout/check out/

(looks like git cmd line as is 😉)

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+
+![](/img/blog-images/2018-04-22-essurvey/spain_stf-1.png)
+
+Looks like Spaniards are very unhappy with their government!
```
If I understand correctly, ESS1 to ESS7 are data collected from surveys every two years (can't recall where I inferred this). Is it possible (in the text for ex) to indicate the years or time span? Can you say "Spaniards are increasingly unhappy with their government!"?

stefaniebutland commented on content/blog/2018-04-22-essurvey.md:
```diff
@@ -0,0 +1,337 @@
+Acknowledgments
+---------------
+
+The package was improved greatly thanks to the reviews of [Thomas
```
link to package repo here? 
If you're interested, this is an opportunity to tell people how they could contribute.

cimentadaj commented:
@stefaniebutland I have addressed all of your comments in the last commit. Thanks a lot for the feedback, it made the post much clearer. I reread it but let me know if this is enough or if I've missed something.

stefaniebutland commented:
Thanks for the quick update @cimentadaj. Looks good. We can publish 2018-06-14. Please update the filename, YAML date, links to images, and delete `925` from `topicid` in YAML. I'll add the topicid just before publishing.

I'll tweet about the post from rOpenSci and tag you. Let me know if there are any key messages you want me to convey in the tweets.

Cheers!

cimentadaj commented:
I think that's it. Let me know if something is missing and thanks for everything!
